clvm_tools/cmds.py

```diff
@@ -296,10 +296,6 @@ def launch_tool(args, tool_name, default_stage=0):
         output = "FAIL: %s %s" % (ex, binutils.disassemble(result, keywords))
         return -1
     except Exception as ex:
-        if args.hex:
-            result = assembled_sexp
```

Comment:
note how `assembled_sexp` is only defined if we're NOT passing in the program as hex. So, if anything fails with a hex program, this will cause a python error.

When looking at fixing this, I realized that `result` is not used here. We re-throw the exception and `result` is tossed.